examples/llama2/notebook/llama2_multiep/config_cpu.json

```diff
@@ -65,6 +65,7 @@
                             "priority": 1
                         }
                     ],
+                    "data_config": "wikitext2_train",
```

Comment:
Same for here too. Sorry, I should have been more specific what changes were needed. for this directory, the only changes needed were the dict -> list which were missed in the previous PR.

Reply:
Fixed.